src/components/AddProblemToSection/AddProblemToSection.tsx

```diff
@@ -1,43 +1,32 @@
-import React, { FC, useState } from 'react';
-import { Button, Text, ModalPreview, ModalHeader, ModalContent } from '@taskany/bricks/harmony';
+import React, { FC } from 'react';
+import { Button } from '@taskany/bricks/harmony';
 import { IconPlusCircleOutline } from '@taskany/icons';
+import { nullable } from '@taskany/bricks';
 
-import { ProblemList } from '../ProblemList/ProblemList';
-import { ProblemFilterBar } from '../ProblemFilterBar/ProblemFilterBar';
+import { usePreviewContext } from '../../contexts/previewContext';
 
 import { tr } from './AddProblemToSection.i18n';
-import s from './AddProblemToSection.module.css';
 
 interface AddProblemToSectionProps {
-    interviewId: number;
-    sectionId: number;
+    problemPreview?: {
+        interviewId?: number;
```

Comment:
interviewId and sectionId should not be optional, since problemPreview is already optional